packages/react-table/scripts/createSubpaths.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Re-export subpath modules into the package root directory for ease of access.
+ */
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = require('fs');
+const { resolve, join } = require('path');
+
+const isIndex = (fileName) => fileName.match(/^index\./);
+
+const updatePaths = (filePath, subPathName) => {
+  const contents = readFileSync(filePath, { encoding: 'utf8' });
+  if (filePath.includes('.map')) {
+    return contents.replaceAll('../../../', `../`);
```

Comment:
There are operations in this file that require explanation IMO. For instance, with this particular condition for `.map` filePaths, why do we need to do this?

Reply:
What is happening and why isn't clear to me, so if I am to ignore the readability of this and expect the logic to change, validity at this point for me would be proof that this accomplishes what we want, and I'll pose that question to you. Meaning, have you tested this out with other packages locally?

Reply:
In my local testing existing imports from /deprecated and /next seem to still work properly (as can be seen in the [OptionsMenu](https://patternfly-react-pr-8848.surge.sh/components/options-menu) and [Select next](https://patternfly-react-pr-8848.surge.sh/components/select/react-next) docs pages in this PRs preview) 

Without this change the deprecated wizard with drawer example is broken when trying to import from just /deprecated, as demonstrated below:
https://user-images.githubusercontent.com/7115053/227536041-57bb9c78-8254-412a-84aa-97d245df1b8a.mov

And with this change:
https://user-images.githubusercontent.com/7115053/227539676-a7df08f2-44da-4f11-9df5-f1ed47e8bb23.mov

Based on this testing I _think_ everything is good to go functionality wise, but since I don't think I fully understand the reasoning behind the approach you went with previously I figured it would be best to double check with you.

Reply:
I went ahead and tested this out against a sample project to make sure /deprecated and /next components can still be imported from `@patternfly/react-core/next` and `@patternfly/react-core/deprecated` (which was the point of the change in the first place). Just testing within this repo is less reliable IMO, but certainly the documentation working as expected is a good sign.